stdlib/public/SwiftShims/NSErrorShims.h

```diff
@@ -15,7 +15,9 @@
 NS_BEGIN_DECLS
 
 NS_INLINE void __NSErrorPerformRecoverySelector(_Nullable id delegate, SEL selector, BOOL success, void *_Nullable contextInfo) {
-    objc_msgSend(delegate, selector, success, contextInfo);
+    void (*msg)(_Nullable id, SEL, BOOL, void* _Nullable) =
+        (void(*)(_Nullable id, SEL, BOOL, void* _Nullable))objc_msgSend;
+    msg(delegate, selector, success, contextInfo);
```

Comment:
Looks pretty subtle to me.  Doesn't this change deserve an explanatory comment in the code?